MCprep_addon/util_operators.py

```diff
@@ -86,6 +86,10 @@ def execute(self, context):
 
 			# make it solid mode, regardless of cycles or eevee
 			context.scene.display.shading.type = 'SOLID'
+
+		# TODO: Confirm this works across all versions of Blender
```

Comment:
Just confirmed this works down to at least blender 2.79 (even if Blender Render engine is active; blender 2.78 is probably also fine with it)

Reply:
(We can remove this comment since this has been confirmed now)